_posts/en/newsletters/2020-04-29-newsletter.md

```diff
@@ -0,0 +1,288 @@
+      bumped][topic cpfp] using [CPFP carve-out][topic cpfp carve out].
+      This would require those transactions to be larger (increasing
+      onchain fees) and presigned (reducing flexibility).  This would
+      only directly affect channels which are unilaterally closed while payments
+      are pending, which is already a situation that that can
+      significantly increase onchain costs and so is something users try
+      to avoid.  However, raising the cost of onchain enforcement also
+      raises the minimum value of payments that can be sent trustlessly
+      through LN with economical enforcement.  Depsite these challenges,
+      as of this writing, this appears to be the most preferred
+      solution.
+
+    Corallo labeled this a severe issue but noted its similar
+    consequences to another known issue related to fee management in
+    onchain LN transactions.  The existing issue (described in
+    [Newsletter #78][news78 anchor outputs]) is that commitment
+    transactions have their feerate set at the time the transaction is
+    signed, which might be days or weeks before they're broadcast to
+    Bitcoin relay nodes.  If the minimum feerate necessary to get a
+    transaction included in the next few blocks has increased
+    significantly since the transaction was last signed, then the
+    commitment transaction might not confirm until after Alice is able
+    to reclaim her funds from Bob, again creating the opportunity for
+    Bob to end up both paying Mallory and giving a refund to Alice.
+    (This existing issue is what developers were working on fixing when
+    the new issue was discovered.  Its ultimate solution also depends on
+    full nodes being able to perform [package relay][Bitcoin Core
+    #14895].)
+
+    So far we're unaware of any real-world losses due to onchain fee
+    management problems in LN, possibly in part because the past two
+    years has seen few large fee spikes that lasted long enough to
+    significantly delay the confirmation of transactions with
+    previously acceptable feerates.  That good luck is unlikely to
+    continue indefinitely, so this new problem gives LN developers an
+    additional reason to prioritize the implementation of improved
+    onchain fee management.  In the interim, node operators concerned
+    about the attack may wish to increase their [cltv_expiry_delta][]
+    to give preimage settlement transactions more time to confirm.
+    Current defaults in popular LN nodes are [14][cl ced] for
+    C-Lightning, [40][lnd ced] for LND, [72][rl ced] for Rust-Lightning,
+    and [144][eclair ced] for Eclair.  Note that increasing the value
+    will make your channels less desirable to spenders who are choosing
+    routes for their payments as higher values increase the worst case
+    amount of time a payment could be stuck waiting to be settled.
+
+- **Multiparty vault architecture:** Antoine "Darosior" Poinsot
```

Comment:
Should the co-author (and original creator, iiuc), Kevin Loaec (spelling?) be named as well?

Reply:
The email seems to be a request for feedback from Dariosior only, so that's the only person I mention.

> Kevin Loaec and I have been working on a new multiparty vault architecture and **I think** it reached the point where we’d welcome some feedback.  (emphasis added)

Reply:
I see your point but it begins by stating both names announcing a joint work; perhaps insert "by [both names]" after "prototype"